mkosi/__init__.py

```diff
@@ -36,6 +36,7 @@
 import urllib.parse
 import urllib.request
 import uuid
+
```

Comment:
I think isort is complaining about this empty line.

Reply:
Applied, except for "`root`" → it shouldn't be quoted, because it's a name, not a path.